auraed/src/lib.rs

```diff
@@ -230,6 +230,10 @@ impl AuraedRuntime {
         let discovery_service_server =
             DiscoveryServiceServer::new(discovery_service.clone());
 
+        let graceful_shutdown =
+            graceful_shutdown::GracefulShutdown::new(cell_service);
```

Comment:
should we just pass all services to this by default?

Reply:
That may be the end result, but we may get a nicer structure that ensures the right sequence is enforced on startup and shutdown.